src/Content/Text/BBCode.php

```diff
@@ -906,7 +906,7 @@ function ($match) use ($callback) {
 					$attributes['avatar'] = ProxyUtils::proxifyUrl($attributes['avatar'], false, ProxyUtils::SIZE_THUMB);
 				}
 
-				return $callback($attributes, $author_contact, $match[3], trim($match[1]) != '');
+				return $match[1] . $callback($attributes, $author_contact, $match[3], trim($match[1]) != '');
```

Comment:
That's not totally like it had been before. See the different behaviour concerning adding ```<hr>```.

Reply:
This is the purpose of the 4th parameter of the callback, to alter the output of the share whether there's content before or not.

See https://github.com/friendica/friendica/blob/ee32cf6b92cb7a7794456b42d8f9d0bf0d1d457f/src/Content/Text/BBCode.php#L959-L961

Reply:
Okay. Let's see how this works.